### ISSUE TITLE
Setting up the documentation is a bit obscure

### ISSUE THREAD

**Issue by dalonsoa** [ImperialCollegeLondon/repositoryr#17]
[Setting up the documentation](https://imperialcollegelondon.github.io/repositoryr/#setup-documentation) is a bit obscure. Maybe a link to an external place will help, but a few things to note:
- Saying at the start `devtools::document` is confusing. It seems it is meant to be called there, but it should be not, but rather at the end of the instructions.
- The Oxygen skeleton only works if there's something to document in the file, otherwise it just fails. 
- An "add.R" is mentioned in that section, but nowhere else, so no idea what it is about.

**Comment by SaranjeetKaur** [comment]
Yep -  that should be `devtools::document()` instead and that will only add the initial structure and will work when there is some documentation. Need to rephrase it.
